kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/OperatorMainIT.java

```diff
@@ -61,12 +79,19 @@ static void afterAll() {
     }
 
     @BeforeEach
-    void beforeEach() {
-        operatorMain = new OperatorMain(null);
+    void beforeEach() throws IOException {
+        managementServer = HttpServer.create(new InetSocketAddress("localhost", 0), 10);
+        operatorMain = new OperatorMain(null, managementServer);
     }
 
     @AfterEach
     void afterEach() {
+        if (kafkaProxy != null) {
```

Comment:
maybe we should do what the JOSDK extension is doing and create a test namespace that we can delete in one shot

Reply:
A task for a separate PR.